README.md

```diff
@@ -1,6 +1,6 @@
 # nodejs-testing
 
-Provide integration with VS Code using the [`node:test` runner](https://nodejs.org/api/test.html). Simply install the extension and start running tests, no setup required.[*](#having-problems-read-this)
+Provide integration with VS Code using the [`node:test` runner](https://nodejs.org/api/test.html). Simply install the extension and start running tests, no setup required.[\*](#having-problems-read-this)
```

Comment:
some automatic formatting from prettier - ok or shall I manually undo it?

Reply:
All good